llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -1023,26 +1029,31 @@ defset list<AMDGPUImageDimIntrinsic> AMDGPUImageDimIntrinsics = {
 //////////////////////////////////////////////////////////////////////////
 defset list<AMDGPUImageDimIntrinsic> AMDGPUImageDimAtomicIntrinsics = {
   multiclass AMDGPUImageDimAtomicX<string opmod, list<AMDGPUArg> dataargs,
-                                   int isFloat = 0> {
+                                   int isFloat = 0, int isAny = 0> {
```

Comment:
Instead of adding more `bool` arguments here, why not take a single `LLVMType` argument which callers can set to `llvm_anyint_ty`/`llvm_anyfloat_ty`/`llvm_any_ty`?